ls1intum/Artemis
# `Development`: Update Apollon version

### Checklist
#### General



### Motivation and Context
Update version of Apollon from [v2.11.2](https://github.com/ls1intum/Apollon/releases/tag/v2.11.2) to [v2.11.3](https://github.com/ls1intum/Apollon/releases/tag/v2.11.3)

### Description
This bugfix version of Apollon fixes an issue with long text insertion in Apollon, the problem stated by @TheTrueVester at one of the [Testing Session](https://confluence.ase.in.tum.de/display/ArTEMiS/Testing+Session+11.07.2022) [Table: Findings > Assigned Issue > Issue Nr: 24]

### Steps for testing
As stated in "Steps to reproduce" section of Testing session. 

OR
1. Create a modeling exercise
2. Insert any element in canvas and try setting its name (or attributes/methods) with more than 100 characters.
3. Observe that the application stops taking input once 100 characters are inserted.



##### Changelog: https://github.com/ls1intum/Apollon/releases/tag/v2.11.3
